tdp/cli/commands/plan/test_dag.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from click.testing import CliRunner
+from .dag import dag
+from ..init import init
+
+
+def test_tdp_plan_dag(collection_path, database_dsn_path, vars):
+    args = [
+        "--collection-path",
+        collection_path,
+        "--database-dsn",
+        database_dsn_path,
+        "--vars",
+        vars,
+    ]
```

Comment:
Can you make a utility function that takes as arguments `collection_path`, `database_dsn_path`, etc. who generates this array so that the option names are only written in one place?

Reply:
I tried, but I found it not very usefull. Many tests are taking different args like `database_dsn_path` or `database_dsn_path` or even without. I think it is more clear to keep it like this.